### PR TITLE
[ticket/15554] Simple footer after load js

### PR DESCRIPTION
**Pull request by ankitjain28may**
Signed-off-by: ankitjain28may <ankitjain28may77@gmail.com>

Checklist:

- [ ] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [ ] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15554

### CODE REVIEW

**Comment by hanakin** [comment]
your commit msg is not in the proper format, i am not sure what the signed off thing you seem to be adding is or is coming from but the body of your commit message must contain the ticket it: PHPBB3-15554

**Comment by VSEphpbb** [comment]
There's a reason the SCRIPTS tag is where it is.

https://github.com/phpbb/phpbb/commit/487822d12f9053ad1728f68031595f01293e59e2

**Comment by DavidIQ** [comment]
It’s basically the same spot as in [overall_footer.html](https://github.com/phpbb/phpbb/blob/7e2cde0844b34e68fe696697a2e890156de39b7c/phpBB/styles/prosilver/template/overall_footer.html#L102)

File: phpBB/styles/prosilver/template/simple_footer.html [hunk 1]
```diff
@@ -33,6 +33,8 @@ <h3 class="alert_title"></h3><p class="alert_text"></p>
 <!-- EVENT simple_footer_after -->
 
 {$SCRIPTS}
+]
```
Comment (DavidIQ) [inline phpBB/styles/prosilver/template/simple_footer.html]: This shouldn’t be here.

**Comment by 3D-I** [comment]
I tried to understand the tracker ticket days ago, once has been posted, Well.. the more I investigated the relationships between files/events the more I am sure there is no need of this PR and therefore the tracker ticket is invalid.

My 2€cents

**Pull request closed without merging**